apps/parsley/src/components/LogPane/index.tsx

```diff
@@ -72,14 +95,31 @@ const LogPane: React.FC<LogPaneProps> = ({ rowCount, rowRenderer }) => {
   }, [listRef, performedScroll, settings, processedLogLines]);
 
   return (
-    <PaginatedVirtualList
-      ref={listRef}
-      className={zebraStriping ? zebraStripingStyles : undefined}
-      paginationOffset={200}
-      paginationThreshold={500000}
-      rowCount={rowCount}
-      rowRenderer={rowRenderer}
-    />
+    <>
+      {stickyHeadersEnabled ? (
+        <StickyHeaders
+          onHeightChange={onStickyHeaderHeightChange}
+          stickyHeaders={stickyHeaders}
+        />
+      ) : null}
+      <PaginatedVirtualList
+        ref={listRef}
+        className={zebraStriping ? zebraStripingStyles : undefined}
+        getScrollOffset={
+          stickyHeadersEnabled ? getScrollOffsetForLine : undefined
+        }
+        onRangeChanged={
+          stickyHeadersEnabled
+            ? ({ startIndex }) => updateStickyHeaders(startIndex)
+            : undefined
+        }
+        overscan={stickyHeadersEnabled ? 0 : 300}
```

Comment:
[nit] I might do something like this to organize the sticky headers props

```ts
const stickyHeadersProps = stickyHeadersEnabled ? {
    getScrollOffset: ...
    onRangeChanged: ...
    overscan: ...
  } : {...}
```
and then spread `stickyHeadersProps` on `PaginatedVirtualList`